modules/govuk_jenkins/templates/jobs/deploy_efg.yaml.erb

```diff
@@ -0,0 +1,80 @@
+---
+- scm:
+    name: alphagov-deployment_Deploy_App
+    scm:
+        - git:
+            url: git@github.com:alphagov/govuk-app-deployment.git
+            branches:
+              - master
+            wipe-workspace: true
+
+- job:
+    name: Deploy_EFG
+    display-name: Deploy_EFG
+    project-type: freestyle
+    description: "<% if @environment != 'integration' %><a href=\"http://www.flickr.com/photos/fatty/9158066939/\">\r\n  <img src=\"https://farm3.staticflickr.com/2835/9158066939_374360ed56_n.jpg\">\r\n</a>\r\n<% end %><h2>You can monitor the application health using the <a href=\"https://grafana.<%= @app_domain %>/#/dashboard/file/application_http_error_codes.json\">4XX and 5XX status dashboard</a></h2>\r\n"
+    <%- if @auth_token -%>
+    auth-token: <%= @auth_token %>
+    <%- end -%>
+    properties:
+        - github:
+            url: https://github.com/alphagov/govuk-app-deployment
+    scm:
+      - alphagov-deployment_Deploy_App
+    builders:
+        - shell: |
+            #!/usr/bin/env bash
+            export DEPLOY_TO="<%= @environment -%>"
+            export DEPLOY_TASK="$DEPLOY_TASK"
+            export TAG="$TAG"
+            export ORGANISATION="<%= @environment -%>"
+            export CI_DEPLOY_JENKINS_API_KEY="<%= @ci_deploy_jenkins_api_key -%>"
+
+            if [ "$DEPLOY_FROM_GITHUB_ENTERPRISE" == "true" ]; then
+              export GIT_ORIGIN_PREFIX="git@github.gds:gds-production-backup"
+            fi
+
+            ./jenkins.sh
+        - trigger-parameterized-builds:
+            - project: Deploy_EFG
+              predefined-parameters: |
+                TARGET_APPLICATION=EFG_TRAINING
+                TARGET_APPLICATION_GIT_REPO=$WORKSPACE/$TARGET_APPLICATION
+              condition: 'UNSTABLE_OR_BETTER'
```

Comment:
I wonder if we should only trigger a deploy of efg-training after a `SUCCESS` status rather than `UNSTABLE_OR_BETTER`